src/colorpicker.cpp

```diff
@@ -17,12 +17,32 @@ static constexpr int GRAB_RADIUS = 16;
 static constexpr int GRAB_SIZE = GRAB_RADIUS * 2 + 1;
 static constexpr int MAGNIFY = 4;
 
+static QScreen *findScreenAt(const QPoint &pos)
+{
+    for (QScreen *screen : QGuiApplication::screens()) {
+        if (screen->geometry().contains(pos))
+            return screen;
+    }
+    return nullptr;
+}
+
+static qreal scaleFactor()
+{
+    QScreen *screen = findScreenAt(QCursor::pos());
+    if (screen)
+        return screen->devicePixelRatio();
```

Comment:
style: always use brackets, even for single line blocks.

```suggestion
    if (screen) {
        return screen->devicePixelRatio();
    }
```

Reminds me I need to setup clang-format for this project.

Reply:
that would be great, do you have already a configuration or can I paste you one for reference?